### PR TITLE
Update base image in .circleci dockerfile

### PR DESCRIPTION
**Pull request by bhavya01**

### CODE REVIEW

File: .circleci/docker/cloudbuild.yaml [hunk 1]
```diff
@@ -6,15 +6,15 @@ steps:
   args: [
```
Comment (vanbasten23) [inline .circleci/docker/cloudbuild.yaml:6]: Where is this file used?

Comment (bhavya01) [reply .circleci/docker/cloudbuild.yaml:6]: I used to create a manual trigger on GCP to build the image. I don't think it is used anywhere else

Comment (vanbasten23) [reply .circleci/docker/cloudbuild.yaml:6]: I was wondering why we need to change this file.

Comment (vanbasten23) [reply .circleci/docker/cloudbuild.yaml:6]: If it's not used, you can drop the change in this file.

**Comment by vanbasten23** [comment]
Let me know if the pr is ready for review

**Pull request closed without merging**